### DISCUSSION TITLE
游戏是不是密钥过期了？

### DISCUSSION THREAD

**Discussion by Wu138dihao138** [pjy612/SteamManifestCache#788] [Question / Answer]
![image](https://github.com/user-attachments/assets/4533362f-eba5-4da0-a690-661cad9e662e)
我无法入库，清单有，无密钥，steamtools与入库助手都开了，冲突了吗？

**Comment by pjy612** [comment]
显示没有密钥 就 自行根据 wiki 预先填充密钥后 使用。

**Reply by Wu138dihao138** [reply]
额....大佬是我要买退离吗？还是找一个共享账号获取密钥？但都没人分享啊？我哪来的密钥？ @pjy612

**Reply by ZEERDEER** [reply]
购买，然后你来分享

**Reply by pjy612** [reply]
购买，然后你来分享
